zh-cn/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>不要为了提问而提问</title>
+  <meta property="og:title" content="Don't ask to ask, just ask" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="zh_CN" />
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    html {
+      height: 100%;
+    }
+    
+    body {
+      min-height: 100%;
+      display: grid;
+      grid-template-rows: 1fr auto;
+
+      margin: 0 auto;
+      max-width: 672px;
+      line-height: 1.6;
+      font-size: 18px;
+      color: #222;
+      background-color: #FAFAFA;
+      padding: 0 10px;
+      font-family: "Open Sans";
+    }
+
+    @media screen and (min-width: 680px) {
+      main {
+        margin: 40px 0;
+      }
+    }
+
+    h1, h2 {
+      line-height: 1.2;
+    }
+
+    footer {
+      grid-row-start: 2;
+      grid-row-end: 3;
+      color: #888;
+      font-size: 9px;
+    }
+
+    a {
+      color: #07A;
+    }
+    a:visited {
+      color: #941352;
+    }
+    footer > a {
+      color: #888 !important;
+    }
+
+    blockquote {
+      background: #F9F9F9;
+      border-left: 10px solid #CCC;
+      margin: 1.5em 10px;
+      padding: 0.5em 10px;
+
+      font-family: monospace;
+    }
+    blockquote p {
+      display: inline;
+    }
+
+    @media screen and (min-width: 400px) {
+      blockquote {
+        display: table;
+      }
+      blockquote > * {
+        display: table-cell;
+      }
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
```

Comment:
The CSS was recently moved to a separate file to reduce duplication

```suggestion
  <link rel="stylesheet" type="text/css" href="/style.css">
```